src/index.js

```diff
@@ -3,6 +3,13 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+(function setPath() {
+  process.env.PATH = [
+    `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+  ].join(path.delimiter);
+})();
 
```

Comment:
I'm not sure if setting a path on require is a good idea. What if user will use only `call` function in its own library and use some child_process functions in a different place ? This put's a risk to inject some unexpected behaviours, I would propose to move it to the call function.